[2023-03-22T15:58:26.493-0400] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: api_dag.is_api_active manual__2023-03-22T19:06:03.876863+00:00 [queued]>
[2023-03-22T15:58:26.504-0400] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: api_dag.is_api_active manual__2023-03-22T19:06:03.876863+00:00 [queued]>
[2023-03-22T15:58:26.504-0400] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-22T15:58:26.504-0400] {taskinstance.py:1280} INFO - Starting attempt 6 of 1
[2023-03-22T15:58:26.504-0400] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-22T15:58:26.508-0400] {taskinstance.py:1300} INFO - Executing <Task(HttpSensor): is_api_active> on 2023-03-22 19:06:03.876863+00:00
[2023-03-22T15:58:26.511-0400] {standard_task_runner.py:55} INFO - Started process 74049 to run task
[2023-03-22T15:58:26.513-0400] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'api_dag', 'is_api_active', 'manual__2023-03-22T19:06:03.876863+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/weatherapi_test.py', '--cfg-path', '/var/folders/71/zhs_yc7n05l7c97rnhn92bb80000gn/T/tmpt9vr21nb']
[2023-03-22T15:58:26.514-0400] {standard_task_runner.py:83} INFO - Job 26: Subtask is_api_active
[2023-03-22T15:58:26.547-0400] {task_command.py:388} INFO - Running <TaskInstance: api_dag.is_api_active manual__2023-03-22T19:06:03.876863+00:00 [running]> on host sunandas-laptop.local
[2023-03-22T15:58:26.584-0400] {taskinstance.py:1507} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=api_dag
AIRFLOW_CTX_TASK_ID=is_api_active
AIRFLOW_CTX_EXECUTION_DATE=2023-03-22T19:06:03.876863+00:00
AIRFLOW_CTX_TRY_NUMBER=6
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-22T19:06:03.876863+00:00
[2023-03-22T15:58:26.586-0400] {http.py:122} INFO - Poking: weather?q=Monroe,GA,US&units=standard&appid=94ca56afa3eee20f7100985fdfe682aa
[2023-03-22T15:58:26.591-0400] {base.py:73} INFO - Using connection ID 'my_api_connection' for task execution.
[2023-03-22T16:11:11.659-0400] {process_utils.py:129} INFO - Sending Signals.SIGTERM to group 74049. PIDs of all processes in the group: [74049]
[2023-03-22T16:11:11.661-0400] {process_utils.py:84} INFO - Sending the signal Signals.SIGTERM to group 74049
[2023-03-22T16:11:19.504-0400] {taskinstance.py:1073} INFO - Dependencies not met for <TaskInstance: api_dag.is_api_active manual__2023-03-22T19:06:03.876863+00:00 [running]>, dependency 'Task Instance Not Running' FAILED: Task is in the running state
[2023-03-22T16:11:19.511-0400] {taskinstance.py:1073} INFO - Dependencies not met for <TaskInstance: api_dag.is_api_active manual__2023-03-22T19:06:03.876863+00:00 [running]>, dependency 'Task Instance State' FAILED: Task is in the 'running' state.
[2023-03-22T16:11:19.511-0400] {local_task_job.py:147} INFO - Task is not able to be run
